### DISCUSSION TITLE
Why is netbox so against feature request?

### DISCUSSION THREAD

**Discussion by peterbaumert** [netbox-community/netbox#12720] [General]
Mainly the lead developer is very negative and opinionated about his own ideas and everyone else seems to be wrong.

You should consider more what the community wants and learn some social skills...

**Comment by adparis99** [comment]
I haven't submitted any ideas myself for a couple of reasons, the foremost being I don't have much actual programming experience and am not familiar enough to know the complexity of a given implementation. But from what I've seen the entire team does a good job of responding to the community and adding features at a reasonable pace, especially those that are highly requested (assuming they fall within the scope of the project).

**Comment by jeremystretch** [comment]
If you're unhappy with how the project is run, you are free to [fork it](https://github.com/netbox-community/netbox/fork) and be responsible for whatever changes you'd like to make, but this is not a forum to complain that your proposals were not accepted.
